hypothesis-python/src/hypothesis/extra/array_api.py

```diff
@@ -269,6 +287,30 @@ def check_valid_minmax(prefix, val, info_obj):
             kw["exclude_max"] = exclude_max
 
         return st.floats(width=finfo.bits, **kw)
+    else:
+        try:
+            float32 = xp.float32
+            float64 = xp.float64
+            complex64 = xp.complex64
+            complex128 = xp.complex64
+        except AttributeError:
+            raise NotImplementedError() from e  # TODO
+        else:
+            kw = {
+                "min_value": min_value,
+                "max_value": max_value,
+                "allow_nan": allow_nan,
+                "allow_infinity": allow_infinity,
+                "allow_subnormal": allow_subnormal,
+                "exclude_min": exclude_min,
+                "exclude_max": exclude_max,
```

Comment:
min_value, max_value, exclude_min, and exclude_max are not applicable to complex numbers; so we should drop them here (and in the tests).